FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -41,6 +42,10 @@
 /// Timeout value for waiting on a fetch response.
 static NSString *const kRemoteConfigFetchTimeoutKey = @"_rcn_fetch_timeout";
 
+/// Listener for the get methods.
+typedef void (^FIRRemoteConfigListener)(NSString *_Nonnull, NSDictionary *_Nonnull)
+    NS_SWIFT_NAME(FIRRemoteConfigListener);
```

Comment:
Remove `NS_SWIFT_NAME`. APIs exposed to Swift should not use typedefs and this looks like its only for internal APIs anyway

Reply:
Done.